src/CommandLineArgumentsParser/Arguments/ValueArgument.cs

```diff
@@ -110,8 +110,12 @@ public TValue Value
         /// </summary>
         public TValue DefaultValue { get; set; }
 
+        public TValue StrongDefaultValue { get; set; }
```

Comment:
Suggestion would be use Forced instead of Strong:
```suggestion
        public TValue ForcedDefaultValue { get; set; }
```